bimsb/packages/staging.scm

```diff
@@ -1587,6 +1587,15 @@ downstream analysis with sleuth.")
                (base32
                 "1zk9y8llv8s751lc37smzlk8q8f980l67j3pmnxqmmw9f4v1lm51"))))
     (build-system r-build-system)
+    (arguments
+     `(#:phases
+       (modify-phases %standard-phases
+         (add-after 'unpack 'patch-NAMESPACE
+           (lambda _
+             (substitute* "NAMESPACE"
+               (("^importFrom\\(rhdf5,h5write\\.default\\)" line)
+                (string-append "#" line)))
+             #t)))))
```

Comment:
FYI: we no longer need #T at the end of build phases.  On the `master` branch it still prints a warning, but on core-updates-frozen (to be merged today, hopefully) it's all fine.

Long ago we used booleans to indicate success or failure for build phases.  Then we switched to using exceptions for failures.  With `core-updates-frozen` the build systems are finally fully aware of this change.

Reply:
Could you please add a link to the upstream discussion above this build phase.  It's always good to keep a record right in the code.

Reply:
Thanks for the feedback. Much appreciated. I tried to address both points. Please double-check though since I did not test actually building `sleuth` using this latest  iteration. :wink: